shared/chat/conversation/header/container.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as Constants from '../../../constants/chat'
+import * as Creators from '../../../actions/chat/creators'
+import Header from '.'
+import {List} from 'immutable'
+import {compose} from 'recompose'
+import {connect} from 'react-redux'
+import {createSelector} from 'reselect'
+import {onUserClick} from '../../../actions/profile'
+
+import type {TypedState} from '../../../constants/reducer'
+
+type OwnProps = {
+  onBack: () => void,
+  onToggleSidePanel: () => void,
+  selectedConversation: ?Constants.ConversationIDKey,
+  sidePanelOpen: boolean,
+}
+
+const getYou = (state: TypedState) => state.config.username || ''
+const getFollowingMap = (state: TypedState) => state.config.following
+const getMetaDataMap = (state: TypedState) => state.chat.get('metaData')
+const getSelectedConversation = (_, selectedConversation: ?Constants.ConversationIDKey) => selectedConversation
+const getSelectedInbox = (state: TypedState, selectedConversation: ?Constants.ConversationIDKey) => (
+  state.chat.get('inbox').find(inbox => inbox.get('conversationIDKey') === selectedConversation)
+)
+
+const getTLF = createSelector(
```

Comment:
returning the tlf as its easy to compare. Arrays/Lists/etc won't follow simple equality checks since we're doing filtering and whatnot